3p/nameframe.max.html

```diff
@@ -3,26 +3,53 @@
 <head>
   <meta charset="utf-8">
   <script>
-    window.onload = function() {
+    var name_data;
+    var writeInBody = false;
+    var instantLoad = false;
+
+    var tryProtected = function(blockFn) {
```

Comment:
Not sure this saves you that much.  Instead you can just wrap your portions below in try/catch and then have a standard error handler.  Something like:
var errorHandler = function() {
    window.name = '';
    window.parent.postMessage(...);
}
try {
   ....
} catch (e) { errorHandler(); }